src/Wrapper.tsx

```diff
@@ -361,8 +352,8 @@ export default function Wrapper({ title, page, children }: { title: any; page: s
         <Box
           component='main'
           sx={{
-            backgroundColor: (theme) =>
-              theme.palette.mode === 'light' ? theme.palette.grey[100] : theme.palette.grey[900],
+            backgroundColor: (curTheme) =>
```

Comment:
You can just remove the function param and use the `theme` var that exists above